lib/fastlane/plugin/android_versioning_kts/actions/get_value_from_build.rb

```diff
@@ -17,14 +17,35 @@ def self.run(params)
             throw "#{type} is not valid type"
         end
 
+        flavor = params[:flavor]
+        flavorSpecified = !(flavor.nil? or flavor.empty?)
+        regex_flavor = Regexp.new(/[ \t]create\("#{flavor}"\)[ \t]/)
```

Comment:
How about this?
```suggestion
        regex_flavor = Regexp.new(/create\("#{flavor}"\)/)
```

Because create is used in many places in gradle, `regex_flavor` looks hacky, but this project is not made to handle many exceptions, it looks good to me.